src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -76,157 +114,520 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
+                              <Typography
+                                sx={{
+                                  alignItems: 'flex-start',
+                                  display: 'flex',
+                                }}
+                                variant="body2"
+                              >
+                                {messages.eventOverviewCard.noLocation()}
+                              </Typography>
+                            </Box>
+                          );
+                        }
+                      }}
+                      value={locationId || ''}
+                    />
+                  </Grid>
+
+                  <Grid item mt={2}>
+                    <ZUIPreviewableInput
+                      {...previewableProps}
+                      renderInput={(props) => (
+                        <TextField
+                          fullWidth
+                          inputProps={props}
+                          label={messages.eventOverviewCard.url()}
+                          onChange={(ev) => setLink(ev.target.value)}
+                          sx={{ marginBottom: 2 }}
+                          value={link}
+                        />
+                      )}
+                      renderPreview={() => {
+                        if (eventData.url && eventData.url !== '') {
+                          return (
+                            <Box ml={4}>
+                              <Typography
+                                color={theme.palette.text.secondary}
+                                component="h3"
+                                variant="subtitle1"
+                              >
+                                {messages.eventOverviewCard.url().toUpperCase()}
+                              </Typography>
+                              <Typography
+                                sx={{
+                                  alignItems: 'flex-start',
+                                  display: 'flex',
+                                }}
+                                variant="body2"
+                              >
+                                {eventData.url}
+                                <Link
+                                  href={getWorkingUrl(eventData.url)}
+                                  sx={{ marginLeft: '8px' }}
+                                  target="_blank"
+                                >
+                                  <OpenInNewIcon color="primary" />
+                                </Link>
+                              </Typography>
+                            </Box>
+                          );
+                        } else {
+                          return <></>;
+                        }
+                      }}
+                      value={link || ''}
+                    />
+                  </Grid>
+                </Grid>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Box>
```

Comment:
Maybe you could even put the margin here? Or on the container that contains all of the card contents (in which case it would probably be `padding` instead)?